packages/language-server/src/xml-view-diagnostics.ts

```diff
@@ -84,7 +88,7 @@ function validationIssuesToLspDiagnostics(
           relatedInformation: map(
             (currIssue as NonUniqueIDIssue).identicalIDsRanges,
             (_) => ({
-              message: "identical ID also used here",
+              message: validations.NON_UNIQUE_ID.relatedInfo ?? "",
```

Comment:
Why are we using a nullish coalescing operator `??` here?
Is this due to Type system limitations? when would we ever pick the `""` branch?